dsl/views.go

```diff
@@ -1451,37 +1464,39 @@ var DefaultNodeSeparation = 600
 // DefaultEdgeSeparation sets the default edge separation for auto layout.
 var DefaultEdgeSeparation = 200
 
-// AutoLayout enables automatic layout mode for the diagram. The
-// first argument indicates the rank direction, it must be one of
+// AutoLayout enables automatic layout mode for the diagram.
+//
+// The First argument indicates the rank direction, it must be one of
 // RankTopBottom, RankBottomTop, RankLeftRight or RankRightLeft
 //
 // AutoLayout must appear in SystemLandscapeView, SystemContextView,
 // ContainerView, ComponentView, DynamicView or DeploymentView.
 //
-// AutoLayout accepts one or two arguments: the layout rank direction and
+// AutoLayout accepts two or three arguments: the layout rank direction and
 // an optional function DSL that describes the layout properties.
 //
 // Example:
 //
-//	var _ = Design(func() {
-//	    var System = SoftwareSystem("Software System", "My software system.")
-//	    var OtherSystem = SoftwareSystem("Other software System")
-//	    var Customer = Person("Customer", func() {
-//	        External()
-//	        Uses(System, "Sends emails", "SMTP")
-//	    })
-//	    Views(func() {
-//	        SystemContextView(SoftwareSystem, "context", "An overview diagram.", func() {
-//	            AddDefault()
-//	            AutoLayout(RankTopBottom, func() {
-//	                RankSeparation(200)
-//	                NodeSeparation(100)
-//	                EdgeSeparation(10)
-//	                Vertices()
-//	            })
-//	        })
-//	    })
-//	})
+//		var _ = Design(func() {
```

Comment:
The indentation seems a bit wacky :) We should probably just keep the original indent depth and simply add the new line.

Reply:
🙈 I think it's the auto formatting of my IDE. Took a while to get it back to not change the whole thing 😰